hazelcast/src/main/java/com/hazelcast/map/impl/MapContainer.java

```diff
@@ -284,6 +286,10 @@ public boolean isInvalidationEnabled() {
     public InterceptorRegistry getInterceptorRegistry() {
         return interceptorRegistry;
     }
+
+    public boolean canBeRemoved() {
```

Comment:
Thanks, completely removed that method, relying on `mapContainers#remove(mapName, mapContainer)` should be sufficient.